sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_helpers.py

```diff
@@ -187,3 +188,16 @@ def convert_to_sub_feedback(feedback):
     if feedback.feedback_type == "Period":
         return PeriodFeedback._from_generated(feedback)  # type: ignore
     raise HttpResponseError("Invalid feedback type returned in the response.")
+
+def convert_datetime(date_time):
+    # type: (Union[str, datetime.datetime]) -> datetime.datetime
+    if not date_time:
+        return date_time
+    if isinstance(date_time, datetime.datetime):
+        return date_time
+    if isinstance(date_time, str):
+        try:
+            return datetime.datetime.strptime(date_time, "%Y-%m-%d")
+        except ValueError:
+            return datetime.datetime.strptime(date_time, "%Y-%m-%d %H:%M:%S")
```

Comment:
If it's a string and doesn't fit either of the formats above, should we just return `date_time` and let service validate?

I think we would want to include (or at least let pass through) the format `"%Y-%m-%dT%H:%M:%SZ"` as that's what the REST API uses/ is sent over the wire and right now that would raise a ValueError